torch/_export/__init__.py

```diff
@@ -785,48 +911,48 @@ def _export(
     if isinstance(f, torch.nn.Module):
```

Comment:
For the lines above related to the `param_buffer_table`, should that also be added to _export_to_non_strict? Since we probably want to keep the param/buffer names the same in both cases right?

Reply:
param_buffer_table is only for dynamo which messes up the state dict. If we're using aot_export_module directly we don't need to do anything special here.

Reply:
Oh ok, then maybe we should move that to export_to_torch_ir (not related to this PR)